packages/core/test/combinator/mergeConcurrently-test.js

```diff
@@ -73,6 +79,11 @@ describe('mergeConcurrently', () => {
 })
 
 describe('mergeMapConcurrently', () => {
+  it('given canonical empty stream, should return canonical empty', () => {
+    const s = mergeMapConcurrently(_ => _, Math.floor(Math.random() * 10), empty())
```

Comment:
Use `id` from `prelude` instead of `_ => _`

Reply:
Done in 76635be